src/platform/site-wide/sass/shame.scss

```diff
@@ -589,3 +589,7 @@ img[data-srcset] {
     width: 100%;
   }
 }
+
+.share-link:hover {
+  color: red;
```

Comment:
I know it's a shame file, but can we use a color variable?

Reply:
this is a draft PR. I'll let you know when it's ready for review.